cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -283,3 +283,54 @@ def _verify_split_test_import(self, target_course_name, source_course_name, spli
         }
 
         self.assertEqual(remapped_verticals, split_test_module.group_id_to_child)
+
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_video_components_present_while_import(self, store):
+        """
+        Test that video components with same edx_video_id are present while re-importing
+        """
+        with modulestore().default_store(store):
+            content_store = contentstore()
+            module_store = modulestore()
+            course_id = module_store.make_course_key('edX', 'test_video_modules', '2015_M4')
+
+            # Import first time
+            import_course_from_xml(
+                module_store,
+                self.user.id,
+                TEST_DATA_DIR,
+                ['test_import_course'],
+                static_content_store=content_store,
+                do_import_static=False,
+                verbose=True,
+                target_id=course_id,
+                create_if_not_present=True
+            )
+
+            # Re-import
+            import_course_from_xml(
+                module_store,
+                self.user.id,
+                TEST_DATA_DIR,
+                ['test_import_course'],
+                static_content_store=content_store,
+                do_import_static=False,
+                verbose=True,
+                target_id=course_id,
+                create_if_not_present=True
+            )
+
+            course = module_store.get_course(course_id)
+            course_key = course.id
+
+            self.assertIsNotNone(course)
+
+            vertical_location = course_key.make_usage_key('vertical', 'vertical_test')
+            vertical = module_store.get_item(vertical_location)
+
+            self.assertIsNotNone(vertical)
+            self.assertGreater(len(vertical.children), 1)
+
+            video = module_store.get_item(vertical.children[1])
```

Comment:
Instead of lines 324-332, you can simply do:

```
self.assertIsNotNone(module_store.get_item(course.id.make_usage_key('video', 'separate_file_video')))
```